src/result_processor.c

```diff
@@ -68,7 +68,7 @@ static int rpidxNext(ResultProcessor *base, SearchResult *res) {
   RPIndexIterator *self = (RPIndexIterator *)base;
   IndexIterator *it = self->iiter;
 
-  if (++self->timeoutLimiter == 100) {
+  if (++self->timeoutLimiter == DEFAULT_TIMEOUT_LIMIT) {
```

Comment:
Why change the count from 100 to 1000? (less accurate timeout, but more efficient?)

Reply:
diff in execution time is minimal to it won't lengthen the timeout by anything significant.
@MeirShpilraien WDYT?